spi/src/main/java/org/eclipse/dataspaceconnector/spi/types/domain/transfer/DataRequest.java

```diff
@@ -215,6 +228,11 @@ public Builder managedResources(boolean value) {
             return this;
         }
 
+        public Builder additionalProperties(Map<String, String> value) {
```

Comment:
Why "additionalProperties" and not just "properties"? Would be more inline with the rest of the code IMO `PROPERTIES_KEY = "dataspaceconnector-properties"`

Reply:
I renamed _additionalProperties_ back to _properties_ as it seems to be good enough, especially that we don't have another properties in DataRequest. I used your suggestion in ArtifactRequestController and extracted a variable called _requestProperty_ . Please take a look if it's looks cleaner now.